### ISSUE TITLE
Feature request: DNS query after connection timeout

### ISSUE THREAD

**Issue by dl1bz** [g4klx/DMRGateway#75]
Can you add an new dns-query if you reconnect after a timeout of dmr-master target? This is very helpful for reconnect if the dmr-master-ip has changed. Remember, not all targets have static IP.
The same is for MMDVMHost, there is an issue with the same request https://github.com/g4klx/MMDVMHost/issues/484 from december 2018 from S57NK with the same content.
Thanks,
73 Heiko, DL1BZ

**Comment by sp2ong** [comment]
I agree that it would help us when we give in Address= the name of the server which has dynamic IP. Currently, I have to script check if the server's IP address has changed and if so do restart dmragteway
